docs/project/reward-algorithm_v2.mdx

```diff
@@ -0,0 +1,52 @@
+---
+id: reward-algorithm_v2
+title: Reward Algorithm_v2
+sidebar_label: Reward Algorithm_v2

```

Comment:
`Reward Algorithm_v2` doesn't look beautiful when it comes to people reading this. Let's rename this to `Reward Algorithm V2` please and remove the underscore and capitalize the `v`.

Reply:
done!